pages/common/animdl.md

```diff
@@ -0,0 +1,36 @@
+# animdl
+
+> A highly efficient, powerful and fast anime scraper.
+> More information: <https://github.com/justfoolingaround/animdl>.
+
+- Download a specific anime:
+
+`animdl download {{anime_title}}`
+
+- Download a specific anime by specifying episode range:
+
+`animdl download {{anime_title}} {{-r|--range}} {{start_episode}}-{{end_episode}}`
+
+- Download a specific anime by specifying download directory:
+
+`animdl download {{anime_title}} {{-d|--download-dir}} {{path/to/download_directory}}`
+
+- Grab the stream URL for a specific anime:
+
+`animdl grab {{anime_title}}`
+
+- Display the anime schedule:
```

Comment:
For some reason when I try to build this, it fails so I can't test it. Can you provide examples on what this outputs? I feel like the wording there feels clunky and adding more context would make it flow better. Like for example `Display the current season anime schedule`

Reply:
Could you please let me know how to build it locally?

Reply:
I tried to use `pip install animdl` in a ven, I tried to use the AUR package and I tried to run 
```
git clone https://www.github.com/justfoolingaround/animdl.git \
&& cd animdl \
&& poetry build \
&& pip install animdl \
&& cd ..
```
as instructed on their github page but all of them got hitched on the same spot

Reply:
Oh you meant `animdl` I thought `tldr`.

Result for `animdl schedule`:
![image](https://github.com/user-attachments/assets/d06c3cf3-3906-4a46-8142-05e40d01eeac)

Reply:
That's super weird. Why is it in reverse order?

Reply:
```suggestion
- Display upcoming anime schedule for the next week:
```

Reply:
> That's super weird. Why is it in reverse order?

No idea, that's how they do it